### DISCUSSION TITLE
PHPB2B V7.0

### DISCUSSION THREAD

**Discussion by ualink** [ulinke/phpb2b#7] [General]
It's a new PHPB2B version for PHP7, hope to help you step into e-commerce!

<hr /><em>This discussion was created from the release <a href='https://github.com/ualink/phpb2b/releases/tag/7.0'>PHPB2B V7.0</a>.</em>

**Comment by chenleigang** [comment]
PB基本已经放弃国内用户啦

**Reply by ualink** [reply]
PHPB2B7.0已经发布，继续免费提供给国内和国外的用户，感谢您的一路关注与支持。

**Reply by zealwebtech** [reply]
Its good to have domestic and foreign users also.

**Reply by ualink** [reply]
nice.
